### PR TITLE
Paths added to messages and message cleanup

### PR DESCRIPTION
**Pull request by pboguslawski**
This mod adds path to error messages. Without it it's difficult for example
to find what path causes permission problems.

Related: https://help.nextcloud.com/t/ocp-files-notpermittedexception-could-not-create-folder/133818
Related: https://help.nextcloud.com/t/could-not-create-folder-false-error/65962/
Author-Change-Id: IB#1121253

### CODE REVIEW

**Review by CarlSchwan — APPROVED** [review]
Make sense to me, thanks!

**Review by kesselb — DISMISSED** [review]
Hi, thanks for your pull request :+1: 

Please rebase this branch to master. Our usual process here is that every change goes to master and is backported (usually by a bot) if necessary.

**Comment by pboguslawski** [comment]
Just force pushed this branch with commit fcb4943071803716d8b7d51c973b440278f18c6d; this commit was created from master and DCO is present now - please change target branch of this PR from `stable24` to `master`.

**Review by miaulalala — APPROVED** [review]

File: lib/private/Files/Node/Folder.php [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?php
 /**
  * @copyright Copyright (c) 2016, ownCloud, Inc.
+ * @copyright Copyright (c) 2022 Informatyka Boguslawski sp. z o.o. sp.k., http://www.ib.pl/
```
Comment (PVince81) [inline lib/private/Files/Node/Folder.php:4]: please don't change the copyright of the whole file

I guess maybe your IDE made this change ?

Comment (PVince81) [inline lib/private/Files/Node/Folder.php:4]:
```suggestion
 * @copyright Copyright (c) 2016, ownCloud, Inc.
```

Comment (pboguslawski) [reply lib/private/Files/Node/Folder.php]: Removed copyright. We add it intentionally to fulfill the AGPL requirement 5a:
https://www.gnu.org/licenses/agpl-3.0.en.html

Comment (tcitworld) [reply lib/private/Files/Node/Folder.php]: You can use the `@author` doctype below instead

Comment (pboguslawski) [reply lib/private/Files/Node/Folder.php]: > You can use the @author doctype below instead

Please point to formal sentence that states that @author way is correct for AGPL. Seems that more standard/common is adding Copyright notice (vide [[1]](https://opensource.google/documentation/reference/copyright), [[2]](https://opensource.stackexchange.com/questions/8639/gpl-v3-0-is-there-a-standard-format-for-modification-notice)).

Comment (tcitworld) [reply lib/private/Files/Node/Folder.php]: I don't make such a claim, this was just a suggestion.

https://github.com/nextcloud/server/blob/master/contribute/HowToApplyALicense.md proves you right anyway.

Comment (PVince81) [reply lib/private/Files/Node/Folder.php:4]: thanks, I wasn't aware of that practice as it seems very rare that people add a second copyright notice line.

then this is fine :+1:

**Comment by PVince81** [comment]
@pboguslawski do you might also adding a signed-off clause as per https://github.com/nextcloud/server/pull/33061/checks?check_run_id=7544272332 ? the DCO part is also stated in https://github.com/nextcloud/server/blob/master/contribute/HowToApplyALicense.md

**Comment by pboguslawski** [comment]
DCO fixed.

**Comment by ChristophWurst** [comment]
Unsure if failing CI is related or not. Could you please rebase to the latest `master` to have another clean CI run?

**Comment by pboguslawski** [comment]
Current master merged to this PR.

**Review by ChristophWurst — APPROVED** [review]

**Comment by ChristophWurst** [comment]
Failing CI unrelated

**Review by come-nc — APPROVED** [review]

**Pull request merged**